packages/shorebird_cli/lib/src/auth/auth.dart

```diff
@@ -248,9 +263,15 @@ class Auth {
   bool get isAuthenticated => _email != null || _token != null;
 
   void _loadCredentials() {
-    final token = platform.environment['SHOREBIRD_TOKEN'];
-    if (token != null) {
-      _token = token;
+    final base64Token = platform.environment[shorebirdTokenEnvVar];
+    if (base64Token != null) {
+      try {
+        _token = CiToken.fromBase64(base64Token);
+      } catch (_) {
```

Comment:
```suggestion
      } catch (_) {
      // TODO(bryanoltman): Remove this legacy behavior after July 2024 or next major release.
      logger.warn('Token failed to parse, assuming an older Google token and re-trying.');
      _token = CiToken(base64Token, googleAuth);
```

Reply:
Not sure if it's worth it, but we could do something like this. 🤷

Reply:
Good call, done. Added more context to the message, along with what steps the user could take to resolve the warning.